src/makeRobot.js

```diff
@@ -38,7 +38,56 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(n) {
+      this.coords.y += n === undefined
+        ? 1 : n > 0
+          ? n : 0;
```

Comment:
1.  please use describetive names -> `steps` insted of `n`
2.  some levels of ternary operators are poor readable, if you have more ten one if-else, so it's better not to use ternary 
3.  can you please simplify this logic? 
  -> it seems you will always have initial values cor this.coords (line 45-48)
  -> couldn't it be just 
```suggestion
     if ( steps> 0) {
         this.coords.y += steps
     }
```

Reply:
Check it now, I have assigned a default value for the steps parameter and used a one level ternary operator
`goForward(steps = 1) {
    this.coords.y += parseInt(steps) > 0 ? steps : 0;

    return this;
},`
Also, I created constants for the evacuate position and added blank line between functions to make code more readable